common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -140,7 +140,9 @@ class InteractForm extends Component<Props, State> {
     );
   }
 
-  private handleInput = name => (ev: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+  private handleInput = (name: any) => (
```

Comment:
`name: string`

Reply:
this throws an error because we dont narrow `string` down to `keyof State`